apps/uci/web/src/components/GetBotList.ts

```diff
@@ -0,0 +1,83 @@
+import { getBotDetailsList } from "@/utils/api-handler";
+import { normalizeUsers } from "@/utils/normalize-user";
+import { reverse, sortBy, without } from "lodash";
+import moment from "moment";
+import { useEffect } from "react";
+import { toast } from "react-hot-toast";
+
```

Comment:
move the types to type folder and use the following naming convention for the types file -> [component-name].d.ts